lumigator/python/mzai/jobs/inference/inference_config.py

```diff
@@ -0,0 +1,39 @@
+from pydantic import BaseModel, ConfigDict
+
+
+class DatasetConfig(BaseModel):
+    path: str
+    model_config = ConfigDict(extra="forbid")
```

Comment:
This field specifies that we do not accept any other parameters in addition to the specified ones. LMK what you think about this, I guess it makes sense to not accept just everything but we could also choose to not break and simply ignore extra stuff.

Reply:
I think we should be strict here - we have so many places we pass around config in the app and so many places it breaks (I know from testing over the past few days 😅 ) that any place we can hammer down these params will be helpful across the codebase.